G-Research.OpenTelemetry.Processor.Partial.Tests/PartialActivityProcessorTests.cs

```diff
@@ -9,29 +10,56 @@ public class PartialActivityProcessorTests
     {
         private List<LogRecord> exportedLogs = [];
         private InMemoryExporter<LogRecord> logExporter;
-        private PartialActivityProcessor processor;
+        private PartialActivityProcessor _processor;
```

Comment:
is this variable renaming relevant to the change?

Reply:
it is, because there is a need for variable name `processor` in other test. according to dotnet naming rules, field variable name should be named with `_` if there is another variable in method that shadows it.

Reply:
i wanted to remove warnings i was getting.